src/internal_events/loki.rs

```diff
@@ -55,14 +54,13 @@ pub struct LokiOutOfOrderEventRewritten;
 
 impl InternalEvent for LokiOutOfOrderEventRewritten {
     fn emit_logs(&self) {
-        warn!(
+        debug!(
             message = "Received out-of-order event, rewriting timestamp.",
             internal_log_rate_secs = 30
         );
     }
 
     fn emit_metrics(&self) {
-        counter!("processing_errors_total", 1,
-                "error_type" => "out_of_order");
+        counter!("rewritten_timestamps_total", 1);
```

Comment:
Not sold on the metric name...

Reply:
Yeah, we don't have a precedent or standard for this. I don't have a strong opinion, other than it should probably have a suffix of `_events_total` (include units).